Automattic/jetpack
# Disable multiple sharing blocks

Fixes https://github.com/Automattic/jetpack/issues/35047

## Proposed changes:

At the moment, we will limit and allow only single sharing block on the page. 
Note: only for Parent Sharing Buttons block, not individual blocks inside

<img width="767" alt="Screenshot 2024-01-22 at 3 40 38 PM" src="https://github.com/Automattic/jetpack/assets/60262784/4ff8bfb4-2dac-4e7a-afcf-59d9d8213667">


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Spin up your site with local changes
 - Make sure you have beta block enabled. In your environment (example Docker) wp-config.php you should have define( 'JETPACK_BLOCKS_VARIATION', 'beta' );
- Add Sharing Buttons block to a post.
- Observe that additional Sharing Buttons blocks can't be added
